--- sync_maildirs.sh	2014-06-12 19:13:26.000000000 +0200
+++ /usr/share/doc/nethserver-mail-server-1.7.0/sync_maildirs.sh	2014-09-11 09:30:59.829861550 +0200
@@ -80,6 +80,7 @@
     # Synchronize maildir:
     /usr/bin/rsync `[ ${DRYRUN} -gt 0 ] && echo '-n -v'` -r -l -t \
 	-e "ssh -p ${sourcePort} -l root" \
+	--delete \
 	"${sourceHost}:/home/e-smith/files/users/${USER}/Maildir/" \
 	"${destinationMaildir}"

