--- ContactsUI.js.old	2015-02-10 14:23:27.438987922 +0100
+++ ContactsUI.js	2015-02-10 14:26:35.456848468 +0100
@@ -22,6 +22,11 @@
         if (searchValue && searchValue.length > 0)
             url += ("&search=" + search["contacts"]["criteria"]
                     + "&value=" + escape(searchValue.utf8encode()));
+// Begin patch for search all by default, dot char
+        else if (currentFolderIsRemote())
+            url += ("&search=name_or_address"
+                    + "&value=.");
+// End of patch
         var sortAttribute = sorting["attribute"];
         if (sortAttribute && sortAttribute.length > 0)
             url += ("&sort=" + sorting["attribute"]
