--- /root/Validator.php 2015-07-01 23:53:54.373659916 +0200
+++ /usr/share/nethesis/Nethgui/System/Validator.php    2015-07-02 00:52:29.183966985 +0200
@@ -717,37 +704,9 @@

     private function evalEmail($value)
     {
-        if (strlen($value) > 254) {
-            $this->addFailureInfo('valid_email,too-long');
-            return FALSE;
-        }
-
-        $parts = explode('@', $value, 2);
-
-        if ( ! isset($parts[0]) || $parts[0] === '') {
-            $this->addFailureInfo('valid_email,missing-localpart');
-            return FALSE;
-        }
-
-        $localPart = $parts[0];
-
-        if (strlen($localPart) > 64 || preg_match('/^[A-Za-z0-9_-](\.?[A-Za-z0-9_-]+)*$/', $localPart) == 0) {
-            $this->addFailureInfo('valid_email,malformed-localpart');
-            return FALSE;
-        }
-
-        if ( ! isset($parts[1])) {
-            $this->addFailureInfo('valid_email,missing-domainpart');
-            return FALSE;
-        }
-
-        $domainPart = $parts[1];
-
-        if ( ! $this->evalHostname($domainPart, 0, PHP_INT_MAX)) {
-            $this->addFailureInfo('valid_email,malformed-domainpart');
-            return FALSE;
-        }
-
+        if ( filter_var($value, FILTER_VALIDATE_EMAIL)==FALSE) {
+           return FALSE;
+       }
         return TRUE;
     }